package/ebs/rancher-ebs

```diff
@@ -323,7 +323,7 @@ delete() {
 
     current_state=$(echo ${volumes} | jq -r '.Volumes[0].State')
     if [ "${current_state}" != "available" ]; then
-        print_error "Failed to delete volume ${VOLUME_ID}, current state: ${current_state} is not available"
+        print_error "Failed to delete volume ${VOLUME_ID}, current state: ${current_state} is not available. Should retry"
```

Comment:
I think the right solution is to poll for the state of available in here and then give up after some amount of time and return an error.

I think you should poll for the state in a loop with an increasing backoff: wait 1 seconds, 2, 4, 8, & max out at trying every 8 seconds for a minute. If it still isn't available, return the error and let the existing agent logic retry 3 times. 
Is 5 minutes a sufficient amount of time for the volume to get